pytest_factoryboy/fixture.py

```diff
@@ -291,6 +291,24 @@ def evaluate(request: SubRequest, value: LazyFixture[T] | T) -> T:
     return value.evaluate(request) if isinstance(value, LazyFixture) else value
 
 
+def noop(*args: Any, **kwargs: Any) -> None:
+    """No-op function."""
+    pass
```

Comment:
NIP: `pass` is not required when docstring is provided

Reply:
ah yeah true that